spaceship.zsh-theme

```diff
@@ -217,6 +217,8 @@ spaceship_ruby_version() {
 
 # Show current vi_mode mode
 spaceship_vi_mode() {
```

Comment:
Please, add check if user don't use `vi-mode`. I've checked this point and that's not OK if everyone will see something like this:

![image](https://cloud.githubusercontent.com/assets/3459374/14561801/8e42c8be-0321-11e6-9223-0dede9a63f9c.png)

Wrap [this](https://github.com/kwinczek/spaceship-zsh-theme/blob/vi_mode/spaceship.zsh-theme#L222-L234) in `if` statement with checking if user use `vi-mode`. Unfortunately, I don't use `vi-mode`, so I can't help you with this. I don't know how to check that.